src/components/Avatar/Avatar.tsx

```diff
@@ -175,6 +162,18 @@ const AvatarStatusItems: FC<BaseAvatarProps> = React.forwardRef(
                 ? { 'aria-label': statusItemProps.ariaLabel }
                 : {})}
             >
+              {statusItemProps.text ? (
+                <span
+                  style={{
+                    fontSize: `calc(2px + ${statusItemProps.size})`,
+                    color: statusItemProps.color,
+                  }}
+                >
+                  {statusItemProps.text}
+                </span>
+              ) : (
+                ''
+              )}
               <Icon {...statusItemProps} />
```

Comment:
The `margin` between the `text` and the `Icon` is not sufficient. They look too close together.

Reply:
No margin was added because the design also does not have it

Reply:
Correct, but kerning is a thing.

Reply:
Maybe like 2px of space? It'll make a big difference.

Reply:
Done